src/item_factory.cpp

```
@@ -1932,15 +1932,32 @@ bool load_min_max(std::pair<T, T> &pa, JsonObject &obj, const std::string &name)
     return result;
 }
 
-bool load_sub_ref(std::unique_ptr<Item_spawn_data> &ptr, JsonObject &obj, const std::string &name)
+bool Item_factory::load_sub_ref( std::unique_ptr<Item_spawn_data> &ptr, JsonObject &obj,
+                                 const std::string &name, const Item_group &parent )
 {
     if (obj.has_member(name)) {
         // TODO!
-    } else if (obj.has_member(name + "-item")) {
+    } else if( obj.has_array( name + "-item" ) || obj.has_array( name + "-group" ) ) {
+        if( name != "contents" ) {
+            debugmsg( "It doesn't make sense to use an array for %s (see doc/ITEM_SPAWN.md)", name.c_str() );
```

I'd throw an exception here.